### ISSUE TITLE
The new version gives error on node_modules

### ISSUE THREAD

**Issue by Hawkeyeanalytics** [FormidableLabs/victory#2426]
I get this error when updated to new version

TypeScript error in /codebuild/output/src057291930/src/buil/node_modules/victory-core/src/victory-animation/util.ts(4,43):
                                 Parameter 'obj' implicitly has an 'any' type.  TS7006
                                 2 | import { isPlainObject, orderBy } from "lodash";
                                 3 |
                                 > 4 | export const isInterpolatable = function (obj) {
                                 |                                           ^
                                 5 |   // d3 turns null into 0 and undefined into NaN, which we don't want.
                                 6 |   if (obj !== null) {
                                 7 |     switch (typeof obj) {

**Comment by Joonas16** [comment]
Same here +1

**Comment by prakash26sep** [comment]
same here +1
Please check this. It's causing the build to crash.

Thanks

**Comment by samuaks** [comment]
same here +1

**Comment by wadehammes** [comment]
same - I have loads of TS errors within Victory packages with latest update

<img width="2168" alt="Screenshot 2022-08-17 at 7 58 33 AM" src="https://user-images.githubusercontent.com/222170/185112968-21847a53-a477-4ff3-b7d6-b1eee52e4dfe.png">

**Comment by frontyard** [comment]
Seeing same issues. Further info if it helps...
I've set version of victory to ~35.5.1
When I do yarn install I get victory version 35.5.3 (CORRECT) **but version 36.6.1 (NOT RIGHT)** of dependecies like victory-core. 
I'm suspecting our issues are related to this unintended major version upgrade.

**Comment by scottrippey** [comment]
Investigating now, thanks for reporting!

**Comment by scottrippey** [comment]
Rolled back latest to `36.6.0`.  Still investigating

**Comment by scottrippey** [comment]
Can anyone post their `tsconfig.json`? I can't reproduce these issues locally

**Comment by wadehammes** [comment]
```
{
  "compilerOptions": {
    "allowJs": true,
    "allowSyntheticDefaultImports": true,
    "baseUrl": ".",
    "declaration": false,
    "esModuleInterop": true,
    "experimentalDecorators": true,
    "forceConsistentCasingInFileNames": true,
    "importHelpers": true,
    "isolatedModules": true,
    "jsx": "preserve",
    "lib": [
      "dom",
      "dom.iterable",
      "esnext"
    ],
    "module": "esnext",
    "moduleResolution": "node",
    "noEmit": true,
    "resolveJsonModule": true,
    "rootDir": ".",
    "skipLibCheck": true,
    "strict": true,
    "target": "es2015",
    "typeRoots": [
      "node_modules/@types"
    ],
    "incremental": true,
    "paths": {
      "@mui/styled-engine": ["./node_modules/@mui/styled-engine-sc"]
    }
  },
  "include": [
    "cssprops.d.ts",
    "next-env.d.ts",
    "**/*.ts",
    "**/*.tsx",
    ".jest/setupTests.ts",
    "i18next.config.ts"
  ],
  "exclude": [
    "node_modules",
  ]
}
```

**Comment by scottrippey** [comment]
Thanks! I'm able to reproduce on an isolated project now.

**Comment by scottrippey** [comment]
Published a fix in `36.6.2`
